Resources/ServerInfo/Guidebook/Antagonist/MinorAntagonists.xml

```diff
@@ -61,6 +61,6 @@
   </Box>
 
   Slimes and spiders have no remarkable features, but will [color=cyan]infest the station[/color] from time to time regardless. Both will give chase and attack anything they see.
-  - Slimes may deal extra [bold]cellular or posion damage[/bold], based upon their color. Water hurts them just as it would hurt a slime person.
-  - Spiders can lay [bold]webs[/bold], which non-spiders have a hard time moving through. They can easily be destroyed with a blade.
+  - Slimes may [bold]deal extra cellular or posion damage[/bold], based upon their color. Water hurts them just as it would hurt a slime person.
```

Comment:
```suggestion
  - Slimes may [bold]deal extra cellular or poison damage[/bold], based upon their color. Water hurts them just as it would hurt a slime person.
```
Minor spelling fix